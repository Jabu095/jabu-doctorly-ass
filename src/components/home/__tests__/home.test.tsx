import { render , screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import HomeComponent from '../home.component';

test('should get home componet', () => {
    render(<HomeComponent />);
    const paragraphElement = screen.getByText('Fuzzy search');
    expect(paragraphElement).toBeInTheDocument();
})

test('renders correctly', () => {
    const tree = renderer
    .create(<HomeComponent />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})