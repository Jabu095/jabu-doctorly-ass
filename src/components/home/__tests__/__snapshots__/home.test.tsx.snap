// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="center"
>
  <h1>
    Fuzzy search
  </h1>
  <form
    onSubmit={[Function]}
  >
    <label>
      Search:
      <input
        onChange={[Function]}
        placeholder="Search..."
        type="text"
        value=""
      />
    </label>
    <button
      className="green-button"
      type="submit"
    >
      Search
    </button>
    <button
      className="red-button"
      onClick={[Function]}
    >
      Delete
    </button>
  </form>
</div>
`;
